config.toml

```diff
@@ -163,21 +163,32 @@ googleAnalytics = ""
   # Call for papers section
   [params.callforpapers]
     enable = true
-    title = "Call For Speakers"
+    title = "Call For Papers"
     bg = false
     CfSpage = "/cardiff-satrday-2019"
       # Go to Organizer > My Events > Dashboard and copy the "CfS Page" string into the `CfSpage` value
     dates = "1 October - 1 November"
 
+
+  # Hackathon section
+  [params.hackathon]
+    enable = true
+    title = "Hackathon"
+    location= """Cardiff University Main Building Council Chamber (Room 1.77, 1st floor, Main Building, Park Place, CF10 3AT)"""
+    bg = false
+    dates = "15 November 2019"
+    times= "13:00-17:30"  
+
```

Comment:
This section could be included, but ideally wouldn't be as we're a little way from being able to deploy the hackathon section. If it's left in, we could live with it, it just wouldn't do anything for a while.